readme.md

```diff
@@ -2,37 +2,34 @@
 
 > Extends [`minimatch.match()`](https://github.com/isaacs/minimatch#minimatchmatchlist-pattern-options) with support for multiple patterns
 
-
 ## Install
 
 ```
 $ npm install multimatch
 ```
 
-
 ## Usage
 
 ```js
-const multimatch = require('multimatch');
+const multimatch = require("multimatch");
```

Comment:
Don't do unrelated changes.

Reply:
my bad, my formatter introduced this. removed it now